Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -27,6 +32,15 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UFUNCTION(BlueprintPure, Category = "C++")
 	FORCEINLINE class UPSHUDComponent* GetProgressionSystemComponent() const { return ProgressionSystemComponentInternal;	}
 
+	/** Returns a current row data */
+	UFUNCTION(BlueprintPure, Category = "C++")
+	FORCEINLINE FPSRowData GetCurrentRowData() const { return SavedProgressionRowDataInternal; }
```

Comment:
constref: `const FORCEINLINE FPSRowData& GetCurrentRowData()`

Reply:
done

Reply:
It's not constref. Right now you just added only 'const', but forgot to add `&`, that means this getter still copying the data on returning. 
Now (wrong):
 `const FORCEINLINE FPSRowData GetCurrentRowData()`
Should be:
`const FORCEINLINE FPSRowData-->&<-- GetCurrentRowData()`

Reply:
changed

Reply:
function was removed as not used